janitor/rabbitmq/publisher.py

```diff
@@ -0,0 +1,73 @@
+import json
+from io import StringIO
+from typing import Any, Dict, Optional, Sequence, cast
+
+from fastavro import json_writer, parse_schema
+from fastavro.types import Schema
+from pika import BasicProperties, BlockingConnection
+
+
+class Publisher:
+    """
+    Basic publisher class for sending messages to RabbitMQ.
+    """
+
+    def __init__(self, connection: BlockingConnection, exchange: str, schema_filepath: str):
+        """Load the schema file ready to publish a message to the specified exchange.
+
+        Arguments:
+            connection {BlockingConnection}: RabbitMQ connection
+            exchange {str}: RabbitMQ exchange to send messages to
+            schema_filepath {str}: filepath to schema file
+        """
+        self._connection = connection
+        self._exchange = exchange
+        self._schema_filepath = schema_filepath
+        self._schema: Optional[Schema] = None
+
```

Comment:
If we use the lab-share-lib library this file wouldnt be needed.
If we don't use it, I would add tests for this file.